### PR TITLE
Fix the key string for TestName

### PR DESCRIPTION
**Pull request by iriszero**
When `test` is used, at least in Ubuntu 18.04 LTS, the line 111-116 failed to parse the `argv` and get the input value.

### CODE REVIEW

File: src/perf/lib/quicmain.cpp [hunk 1]
```diff
@@ -69,7 +69,7 @@ QuicMainStart(
         return QUIC_STATUS_INVALID_PARAMETER;
     }
 
-    const char* TestName = GetValue(argc, argv, "test");
+    const char* TestName = GetValue(argc, argv, "TestName");
```
Comment (nibanks) [inline src/perf/lib/quicmain.cpp:72]: I don't mind having `test` to be an alias to `TestName` here, but we need to support both. For instance, if `TestName` isn't present, then check for `test`.

Comment (iriszero) [reply src/perf/lib/quicmain.cpp:72]: > I don't mind having `test` to be an alias to `TestName` here, but we need to support both. For instance, if `TestName` isn't present, then check for `test`.

That would be better:
a simple short circuit may do?
```
const char* TestName = GetValue(argc, argv, "TestName") || GetValue(argc, argv, "test");
```

Comment (nibanks) [reply src/perf/lib/quicmain.cpp]: Essentially, but I think you'll need separate statements.

Comment (iriszero) [reply src/perf/lib/quicmain.cpp]: @nibanks Please check out 00bd717, linked below.

File: src/perf/lib/quicmain.cpp [hunk 2]
```diff
@@ -69,9 +69,15 @@ QuicMainStart(
         return QUIC_STATUS_INVALID_PARAMETER;
     }
 
-    const char* TestName = GetValue(argc, argv, "test");
-    ServerMode = TestName == nullptr;
-
+    const char* TestName;
+    
+    if ((TestName = GetValue(argc, argv, "TestName")) == nullptr) {
```
Comment (nibanks) [inline src/perf/lib/quicmain.cpp:74]: Instead, simply add a line after 72 to get test if Test name variable is null.

**Review by nibanks — APPROVED** [review]
